x-pack/plugins/apm/server/routes/alerts/rule_types/transaction_duration/register_transaction_duration_rule_type.test.ts

```diff
@@ -64,7 +64,7 @@ describe('registerTransactionDurationRuleType', () => {
       environment: 'development',
       interval: `5 mins`,
       reason:
-        'Avg. latency is 5,500 ms in the last 5 mins for service: opbeans-java, env: development, type: request. Alert when > 3,000 ms.',
+        'Avg. latency is 5.5 s in the last 5 mins for service: opbeans-java, env: development, type: request. Alert when > 3.0 s.',
```

Comment:
Who is the best person to check if these changes are OK? PM/Designer or ...?

Reply:
In APM we would display this as milliseconds (up until 10.000 ms) but if this makes it more consistent with the rest of Observability I think it is fine